core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -397,3 +397,29 @@ oppia.directive('testInteractionPanel', [function() {
     ]
   };
 }]);
+oppia.directive('heightWatcher', [function() {
+  return {
+    restrict: 'A',
+    link: function(scope, element) {
+      scope.$watch(
+      function() {
+        return element[0].offsetHeight;
+      },
+      function(newVal, oldVal) {
+        if (newVal !== oldVal) {
+          if (newVal > 630 - 40) {
```

Comment:
@seanlip how can we create shadow dummy card while we are editor_view.I mean both are in different context.so,I think we cant do like what have been done to know new height to animate to.please correct me if I am going wrong anywhere